install/python_setup.bash

```diff
@@ -40,7 +40,7 @@ mkdir -p $ADT4_ENV
 # Install ROMAN
 ##################################
 if [ "$install_roman" = true ]; then
-    python3 -m venv $ADT4_ENV/roman
+    python3 -m venv $ADT4_ENV/roman --clear
```

Comment:
Is this necessary? Ideally the setup script is idempotent so we can always re-run to ensure stuff is up to date. Does this means it's going to re-download the whole 5GB virtual environment every time I do that?